src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_app_extension.h

```diff
@@ -182,6 +182,36 @@ class RCAppExtension : public application_manager::AppExtension {
    */
   std::set<ModuleUid> InteriorVehicleDataSubscriptions() const;
 
+  /**
+   * @brief AddPendingSubscription add pending subscription
+   * @param module interior data specification(zone, data type)
+   * @return
```

Comment:
@SNiukalov please specify the return value

Reply:
@LitvinenkoIra 
Thanks, fixed in:https://github.com/LuxoftSDL/sdl_core/pull/129/commits/6f71dec232f0b88b3beeacc1c02fdc23af9d6190